packages/field/src/css/field.ts

```diff
@@ -32,12 +32,19 @@ export default {
     color: colorsTextIcon.highOnLight,
     display: 'flex',
     fontWeight: type.fontWeightDefault,
-    padding: layout.spacingSmall,
+    padding: `0 ${layout.spacingMedium}`,
```

Comment:
0 out padding. Use `height` on field to guarantee it. horz padding to match figma.